src/views/ApiTest.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div>
+    <!-- 登録情報を表示 -->
+    <div v-for="content in contents" :key="content.id">
+      <span>{{ content.id }}</span>
+      <span>{{ content.title }}</span>
+      <span>{{ content.body }}</span>
+    </div>
+    <form>
+      <!-- 投稿フォーム -->
+      <span>title</span>
+      <!-- 投稿タイトルをバインド -->
+      <input v-model="posts.title" type="text" />
+      <span>author</span>
+      <!-- 投稿内容をバインド -->
+      <input v-model="posts.author" type="text" />
+    </form>
+    <button type="button" @click="postPosts()">投稿</button>
+  </div>
+</template>
+<script>
+//axiosをインポート
+import axios from "axios";
+
+export default {
+  data() {
+    return {
+      //jsonファイルが持つcontentを表示させる空の配列を作る
+      contents: [],
+      //title,authorを保存させる空のデータを作る
+      posts: {
+        title: "",
+        author: "",
+      },
+    };
+  },
+  //Vueインスタンスのmount後にjsonデータを取得させる
+  mounted() {
+    // axiosを使用
+    this.getContents();
+    // async/awaitを使用
+    // this.getContentsAsync();
+  },
```

Comment:
たぶん非同期処理にしてるので、 `async` 関数にしとかないと2つ以上非同期処理した時に狂う気がします

```suggestion
  async mounted() {
    // axiosを使用
    await this.getContents();
    // async/awaitを使用
    // this.getContentsAsync();
  },
```